test/integration/ovs/ofctrl_test.go

```diff
@@ -240,11 +241,12 @@ func TestOFctrlFlow(t *testing.T) {
 		if err != nil {
 			t.Errorf("Failed to DeleteFlowsByCookie: %v", err)
 		}
-		require.NoError(t, wait.PollImmediate(time.Millisecond*100, time.Second, func() (done bool, err error) {
-			flowList, err = OfctlDumpTableFlowsWithoutName(ovsCtlClient, myTable.GetID())
-			require.Nil(t, err)
-			return len(flowList) == 0, nil
-		}), "Failed to delete flows by CookieID")
+		require.NoError(t, wait.PollUntilContextTimeout(context.Background(), time.Millisecond*100, time.Second, true,
+			func(ctx context.Context) (done bool, err error) {
+				flowList, err = OfctlDumpTableFlowsWithoutName(ovsCtlClient, myTable.GetID())
+				require.Nil(t, err)
```

Comment:
not from this PR, but should be `require.NoError`

same for other tests below

Reply:
Changed all `require.Nil(t, err)` to `require.NoError(t, err)`